e2e/actions.ts

```diff
@@ -123,8 +123,7 @@ export const createActions = ({
     },
   },
   drawer: {
-    close: async () =>
-      await screen.getByRole("button", { name: "Back to main" }).click(),
+    close: async () => await screen.getByTestId("section-close").click(),
```

Comment:
I believe it would be btter to add aria-label="Back to main" on the Box (which should be a Button I guess) so that you can still reach it via `getByRole('button', { name: 'Back to main' })`

If not possible to make it a button, I still believe having an aria-label could be possible, and use getByLabel.

Reply:
I agree (just to note, this is how we treated collapsible section "buttons" before, as `div` elements with an `onClick` evert handler, so at least it's aligned 😅) – will check if it doesn't break things to actually use a button here 👍